### PR TITLE
make doSaveDependingOnFiles nullable again

### PR DESCRIPTION
**Pull request by dfx413**
addresses #23

### CODE REVIEW

File: src/DI/ClassAliasMap.php [hunk 1]
```diff
@@ -3,6 +3,6 @@
 declare(strict_types = 1);
 
 return [
-	\Kdyby\DoctrineCache\NotImplementedException::class 	=> \Kdyby\DoctrineCache\Exception\NotImplementedException::class,
-	\Kdyby\DoctrineCache\Exception::class 					=> \Kdyby\DoctrineCache\Exception\Exception::class,
+	\Kdyby\DoctrineCache\Exception\NotImplementedException::class => \Kdyby\DoctrineCache\Exception\NotImplementedException::class,
```
Comment (VBoss) [inline src/DI/ClassAliasMap.php]: This should not be here. You are aliasing same class name. This is here just to provide backwards compatibility for renamed namespaced class `\Kdyby\DoctrineCache\NotImplementedException`

File: src/DI/ClassAliasMap.php [hunk 2]
```diff
@@ -3,6 +3,6 @@
...
+	\Throwable::class => \Throwable::class,
```
Comment (VBoss) [inline src/DI/ClassAliasMap.php]: Same here

**Review by Spamercz — CHANGES_REQUESTED** [review]
Nullable fix is OK. But class aliases should not be in this. Please remove that so I can merge this.

**Comment by dfx413** [comment]
Hello, sorry for inconvenience. Should be OK now.

**Review by Spamercz — APPROVED** [review]

**Comment by VBoss** [comment]
No worries :). Merging

**Comment by greeny** [comment]
Can this be tagged, please?

**Pull request merged**